builder/ConfTargets.cmake

```diff
@@ -110,6 +110,8 @@ option( MFX_ENABLE_VP9_VIDEO_ENCODE "Enable VP9 encoder?" ON)
 
 option( MFX_ENABLE_MCTF "Build with MCTF support?"  ${MFX_1_25_OPTIONS_ALLOWED} )
 
+set( MFX_PRODUCT_NAME "Intel(R) Media SDK" )
```

Comment:
Rewrite as:
```
if (NOT DEFINED MFX_PRODUCT_NAME)
  set( MFX_PRODUCT_NAME "Intel(R) Media SDK" )
endif()
```
And move it below after `include(${MFX_CONFIG_FILE}) ` and before `configure_file(mfxconfig.h.in mfxconfig.h)`. Since this is not a cmake option(), we can do this and this will avoid redifinition of the variable.